t/01-basic.t

```diff
@@ -341,3 +341,27 @@ OK
 --- no_error_log
 [error]
 [warn]
+
+=== TEST 13: Should return error on broken response status line
+--- http_config eval: $::HttpConfig
+--- config
+    location = /a {
+        content_by_lua_block {
+            local http = require "resty.http"
+            local httpc = http.new()
+            local res, err = httpc:request_uri("http://127.0.0.1:12345")
+
+            assert(err, "there must be an error")
```

Comment:
It would be nice to test for an explicit error message string. Ideally with an additional test so that both error cases are covered.